js/ui/widget/ui.widget.js

```diff
@@ -183,6 +182,11 @@ var Widget = DOMComponentWithTemplate.inherit({
     _init: function() {
         this.callBase();
         this._initContentReadyAction();
+        this._initKeyboardHandledAction();
+    },
+
+    _initKeyboardHandledAction() {
+        this._keyboardHandledAction = this._createActionByOption('onKeyboardHandled');
```

Comment:
Is it necessary to create action here? It is necessary for public event, but it is private.
Action is not light thing. Every new action in base widget decrease performance.
Also by default action has "disabled" and "readOnly" validators. So in readOnly widget callback onKeyboardHandled will not be executed. Is it expected behavior?

Maybe it will be better avoid creating action here. And instead of 
```
this._keyboardHandledAction(options);
```
call callback directly
```
const onKeyboardHandled = this.option("onKeyboardHandled");
onKeyboardHandled && onKeyboardHandled(options);
```